JapaneseFilter/sections/specific.txt

```diff
@@ -2429,7 +2430,7 @@ blog.livedoor.jp##div[id^="shitarss_title_"]
 negisoku.com###main > div.column-inner > div.column-inner-2 > a[href^="https://e-gaming.g2afse.com/click"]
 negisoku.com###rss-box
 negisoku.com##.article-outer-3 > a[href^="https://e-gaming.g2afse.com/click"]
-negisoku.com##.plugin-memo
+nokuo.blog.jp,negisoku.com##.plugin-memo
```

Comment:
`##.plugin-memo` is used by thousands of `blog.jp` sites both for ads and for legitimate contents. To cover as much sites as possible without false positve, remove this line and then change an existing rule in css-extended `sunri2525.blog.jp,akb48rompen.com,hattatu-matome.ldblog.jp,blog.livedoor.jp#?#.plugin-memo > div.side > ins.adsbygoogle:upward(2)` to `blog.jp,akb48rompen.com,ldblog.jp,blog.livedoor.jp#?#.plugin-memo > div.side > ins.adsbygoogle:upward(2)`.